### DISCUSSION TITLE
Verification in order to post comments

### DISCUSSION THREAD

**Discussion by televandalist** [RetroAchievements/RAWeb#1799] [Ideas]
As of now, when a user posts in the forum for the first time, the post is hidden until it is verified by an admin/site mod.

We need to extend this to user and game wall comments.

Honestly, I'd extend it to tickets and site messages, too. Maybe allow the user to contact RAdmin, but that's it?

**Comment by Tsearo** [comment]
To add onto this it might be nice to add a mute user button for admin/mod on the user profile that simply reverts the verification
